### PR TITLE
Implement changing of tab title when privacy mode is enabled

### PR DESCRIPTION
**Pull request by craftyguy**
Per #2697

### CODE REVIEW

File: qutebrowser/config/configdata.py [hunk 1]
```diff
@@ -377,7 +377,7 @@ def data(readonly=False):
              SettingValue(typ.FormatString(fields=['perc', 'perc_raw', 'title',
                                                    'title_sep', 'id',
                                                    'scroll_pos', 'host',
-                                                   'backend']),
+                                                   'backend', 'priv']),
```
Comment (The-Compiler) [inline qutebrowser/config/configdata.py]: Not sure whether it makes sense to abbreviate it - why not simply `private`?

File: qutebrowser/config/configdata.py [hunk 2]
```diff
@@ -389,7 +389,8 @@ def data(readonly=False):
              "* `{id}`: The internal window ID of this window.\n"
              "* `{scroll_pos}`: The page scroll position.\n"
              "* `{host}`: The host of the current web page.\n"
-             "* `{backend}`: Either 'webkit' or 'webengine'"),
+             "* `{backend}`: Either 'webkit' or 'webengine'"
+             "* `{priv}` : Indicates when privacy mode is enabled.\n"),
```
Comment (The-Compiler) [inline qutebrowser/config/configdata.py]: privacy -> private :wink:

**Comment by craftyguy** [comment]
Thanks for the feedback, I've made the changes you requested!

File: qutebrowser/config/configdata.py [hunk 3]
```diff
@@ -389,7 +389,8 @@ def data(readonly=False):
              "* `{id}`: The internal window ID of this window.\n"
              "* `{scroll_pos}`: The page scroll position.\n"
              "* `{host}`: The host of the current web page.\n"
-             "* `{backend}`: Either 'webkit' or 'webengine'"),
+             "* `{backend}`: Either 'webkit' or 'webengine'"
```
Comment (The-Compiler) [inline qutebrowser/config/configdata.py]: Missing newline here, but I'll fix it up when merging :wink:

Comment (craftyguy) [inline qutebrowser/config/configdata.py]: Dang, that's what I get for copying/pasting the 'private line' I had inserted arbitrarily in the middle of that block to the end. Sorry!

**Comment by The-Compiler** [comment]
Thanks!

**Pull request merged**